airflow/www/utils.py

```diff
@@ -282,7 +289,18 @@ def dt(attr):
         f = f.isoformat() if f else ''
         if timezone.utcnow().isoformat()[:4] == f[:4]:
             f = f[5:]
-        return Markup("<nobr>{}</nobr>").format(f)
+        return Markup('<time datetime="{}">{}</time>').format(f, f)
+    return dt
+
+
+def localized_f(attr_name):
```

Comment:
```suggestion
def localized_dt_f(attr_name):
```

Otherwise it's not immediately obvious from the use in views.py what is being localised